hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -245,10 +247,9 @@ public static TypedProperties flinkConf2TypedProperties(Configuration conf) {
   }
 
   public static void checkRequiredProperties(TypedProperties props, List<String> checkPropNames) {
-    checkPropNames.forEach(prop -> {
-      if (!props.containsKey(prop)) {
-        throw new HoodieNotSupportedException("Required property " + prop + " is missing");
-      }
-    });
+    checkPropNames.forEach(prop ->
+        Preconditions.checkState(
```

Comment:
Why can not merge the next two lines into this line?

Reply:
Personally i prefer shorter line code. Refactoring it if Hoodie thinks long line code is ok.